.ci/install.sh

```diff
@@ -2,7 +2,12 @@
 
 set -e
 
-INSTALL_DIR="$HOME/.fnm"
+if [ -d "$HOME/.fnm" ]; then
+  INSTALL_DIR="$HOME/.fnm"
+else
+  INSTALL_DIR="${XDG_DATA_HOME:-$HOME/.local/share}/fnm"
```

Comment:
I wonder if that's enough. MacOS way would be `~/Library/Application Support/fnm`

what do you think? SHould we keep `~/.local/share/fnm` just for the installation process?

Reply:
Oops, thanks for your friendly reminder.

I just thought on MacOS `fnm` would be installed at `~/.local/share/fnm`

Reply:
There’s no convention on Macs for ~/.local to even exist at all/